src/test/java/com/hivemq/mqtt/handler/connect/ConnectHandlerTest.java

```diff
@@ -145,7 +146,7 @@ public void setUp() throws Exception {
                 any(),
                 isNull())).thenReturn(Futures.immediateFuture(null));
 
-        channel = new EmbeddedChannel(new DummyHandler());
+        channel = new EmbeddedChannel(new MyChannelId(), new DummyHandler());
```

Comment:
why do we need the channel id? I guess with one of my comments above we should not need the channel id in our code

Reply:
Yes, this is not required anymore.